src/Dispatcher/Masa.Contrib.Dispatcher.IntegrationEvents.Dapr/Processor/DeleteLocalQueueExpiresProcessor.cs

```diff
@@ -14,11 +16,9 @@ public DeleteLocalQueueExpiresProcessor(IOptions<DispatcherOptions> options)
     /// </summary>
     /// <param name="stoppingToken"></param>
     /// <returns></returns>
-    public override Task ExecuteAsync(CancellationToken stoppingToken)
+    protected override Task ExecutingAsync(CancellationToken stoppingToken)
```

Comment:
Why you need async? Support MQ in the future?

Reply:
No need for async here, it will be removed in a later commit

Reply:
processed